### ISSUE TITLE
Which version of this package should be used?

### ISSUE THREAD

**Issue by cookpete** [BlackBoxVision/react-qr-reader#61]
Hey @JonatanSalas,

It seems like both the original [react-qr-reader](https://github.com/react-qr-reader/react-qr-reader) and this version are now being worked on in parallel. The original repo has [reset the version from `5.0.0` to `3.0.0`](https://github.com/react-qr-reader/react-qr-reader/commit/f82d1f408ec05c6a10d978f29b90da00e465e545#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3) and has recently published `3.0.0-beta-1` to npm, which includes [several updates](https://github.com/react-qr-reader/react-qr-reader/compare/bfd5e3e...e25dd420) that are not present in this repo, which has kept the `5.0.0` version and was published months ago.

Which repo should be used going forward? There is zero visibility of this repo in the original repo (I stumbled across it in an issue) so it is very unclear what the "correct" package to use is. Is there a long term plan to combine the efforts again?

**Comment by manutuero** [comment]
Hello @cookpete, how are you?

After a conversation we had with the creator of the library @JodusNodus, we decided to unify everything in the `react-qr-reader` organization package. Therefore, we will stop maintaining the BlackBox Vision package to join forces and give traction to said organization, that way it is much more efficient than having two libraries that do the same thing and can be outdated. I hope I have helped clarify a bit.

Greetings.

**Comment by cookpete** [comment]
Great, thanks for clarifying @manutuero.

I’ll leave this issue open for visibility, and also maybe as a reminder to make it clear in the README that this repo is now superseded by the other one?

@JodusNodus What is required to get the `3.0.0` release out of beta?

**Comment by manutuero** [comment]
@cookpete Sure!
